css-positioning/fixed.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  top: 70px;
+  left: 260px;
```

Comment:
You can do `width: 100%` to get the same effect without having to push this element big a hug amount using positioning